.github/workflows/e2e.yaml

```diff
@@ -32,9 +28,13 @@ jobs:
       - name: Setup Helm
         uses: fluxcd/pkg/actions/helm@main
       - name: Run tests
-        run: make test
+        uses: docker://golang:1.15-alpine
         env:
           KUBEBUILDER_ASSETS: ${{ github.workspace }}/kubebuilder/bin
+        run: |
+          apk add --no-cache docker build-base git
```

Comment:
I don't think we need docker to run unit tests

Reply:
Yup, copy pasta.

Reply:
Also Git is included in the go image